### DISCUSSION TITLE
请问flinkCDC支持哪些数据？一些国产数据库，比如：GoldenDB、GaussDB、达梦等，是否支持呢？

### DISCUSSION THREAD

**Discussion by chenjiapi** [apache/flink-cdc#3268] [Q&A]
我在网上看到，”支持从 MySQL、MariaDB、RDS MySQL、Aurora MySQL、PolarDB MySQL、PostgreSQL、Oracle、MongoDB、SqlServer、TiDB、OceanBase 等数据库中实时地读取存量历史数据和增量变更数据“。
一些国产数据库，比如：GoldenDB、GaussDB、达梦等，是否支持呢？

**Comment by yuxiqian** [comment]
Hi @chenjiapi, currently you can connect to MySQL, Doris, StarRocks, Kafka, and Paimon in CDC Pipeline jobs with Flink CDC 3.1 and above. Also, you can Db2, MongoDB, MySQL, OceanBase, Oracle, Postgres, SQL Server, TiDB, and Vitess connectors in Flink SQL / DataStream jobs. MariaDB is not supported yet, but PR #2494 is trying to add MariaDB support.

GoldenDB, GaussDB, and DM are not supported yet. If you want more connectors, please open a feature request ticket on [Flink JIRA](https://issues.apache.org/jira/projects/FLINK/summary).
